ci/steps/promote_step.py

```diff
@@ -115,13 +115,18 @@ def promote_step(
     committish: str,
     version: str,
     promote_target: str,
+    manifest_set_key_result: str,
 ):
     cicd_cfg = glci.util.cicd_cfg(cfg_name=cicd_cfg_name)
     flavour_set = glci.util.flavour_set(flavourset)
     flavours = tuple(flavour_set.flavours())
     build_type: glci.model.BuildType = glci.model.BuildType(promote_target)
     build_target_set = glci.model.BuildTarget.set_from_str(build_targets)
 
+    # write result so thet we always have a result:
+    with open(manifest_set_key_result, 'w') as f:
+       pass
```

Comment:
maybe write a placeholder (such as: <actual log output should be here - if you read this, an error occurred during log collection>)?

also, if this is a wellknown case, we might as well check for the file's existence, rather than creating a dummy file..

Reply:
would require special parsing when reading.... not sure as no one will ever read this file